src/Lykke.Tools.AssetMigrator/Program.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Threading.Tasks;
+using Lykke.Tools.AssetMigrator.Implementations;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+
+
+namespace Lykke.Tools.AssetMigrator
+{
+    internal static class Program
+    {
+        
+        private static async Task<int> Main(string[] args)
+        {
+            var resultCode = 0;
+            
+            try
+            {
+                var services = ConfigureServices();
+                var serviceProvider = services.BuildServiceProvider();
+                var rootCommand = serviceProvider.GetService<IRootCommand>();
+                var app = rootCommand.Configure();
+
+                resultCode = app.Execute(args);
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e.Message);
+
+                resultCode = 1;
+            }
+
+            // Flushing logger
+            await Task.Delay(100);
+
+            return resultCode;
+        }
+
+        private static IServiceCollection ConfigureServices()
+        {
+            var serviceCollection = new ServiceCollection();
+
+            serviceCollection.AddLogging
```

Comment:
You could create Lykke ILogFactory with needed loggers as here https://github.com/LykkeCity/Lykke.Service.LoggingAdapter/blob/master/src/Lykke.Service.LoggingAdapter.Services/Log/LogFactoryStorage.cs#L52 this will allow you to use logger with Lykke libraries